### PR TITLE
Geomark is disabled for mobile devices

### PR DESCRIPTION
**Pull request by michaelpnelson**

### CODE REVIEW

**Comment by michaelpnelson** [comment]
Some notes from my testing ...

When an SMK app is created with "mobile" specified, the geomark panel doesn't appear, even when enabled in smk-config.json; I believe this is because geomark is not included in the built app (I don't see it among the node module files).

When an SMK app is created with "default" specified, the geomark panel appears for a mobile device, but there's no functionality when it's selected - we don't enter draw mode and buttons don't work. 

We disable other tools this way (coordinate, zoom, markup, minimap) but this may be the first time we've disabled a panel this way. I didn't see a way to exclude the panel from the list of tools for mobile devices; we test on the smk.$device variable, but it looks like we don't have a handle on the smk object until after tools are initialized.

File: src/smk/tool/geomark/tool-geomark.js [hunk 1]
```diff
@@ -52,6 +52,10 @@ include.module( 'tool-geomark', [
             this.promptReply = '';
         },
         function ( smk ) {
+            if ( smk.$device == 'mobile' ) {
```
Comment (dgboss) [inline src/smk/tool/geomark/tool-geomark.js]: We should use === here.

**Review by dgboss — CHANGES_REQUESTED** [review]

**Review by dgboss — CHANGES_REQUESTED** [review]
Also consider updating documentation to indicate that the geomoark tool isn't supported when device === mobile.

**Comment by michaelpnelson** [comment]
> Also consider updating documentation to indicate that the geomoark tool isn't supported when device === mobile.

Done.

File: src/smk/tool/geomark/tool-geomark.js (hunk 1 above)
Comment (michaelpnelson) [reply src/smk/tool/geomark/tool-geomark.js]: Done.

**Review by dgboss — APPROVED** [review]
Looks good.

**Pull request merged**